README.md

```diff
@@ -67,41 +62,43 @@ If you experience slow launch times of fastlane, try running:
 
     gem cleanup
 
-System Requirements: `fastlane` requires Mac OS X or Linux with Ruby 2.0.0 or above.
+**System Requirements:** `fastlane` requires Mac OS X or Linux with Ruby 2.0.0 or above
 
-If you want to take a look at a project already using `fastlane`, check out the [fastlane-examples](https://github.com/fastlane/examples) with `fastlane` setups by Wikipedia, Product Hunt, MindNode and more.
+If you'd like to take a look at a project already using `fastlane` check out [fastlane-examples](https://github.com/fastlane/examples) which includes `fastlane` setups by Wikipedia, Product Hunt, MindNode, and more.
 
 ## Quick Start
 
-The setup assistant will create all the necessary files for you, using the existing app metadata from iTunes Connect or Google Play.
+`fastlane`'s setup assistant creates all the necessary files to get you started, using existing app metadata from iTunes Connect or Google Play.
 
 - `cd [your_project_folder]`
 - `fastlane init`
-- Follow the setup assistant, which will set up `fastlane` for you
-- Further customise the `Fastfile` with [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md).
+- Follow the assistant's prompts
+- Further customise the Ruby based `Fastfile` with additional [actions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md)
```

Comment:
My intent here was to make clear that it's not only actions that compose a Fastfile.